mne/label.py

```diff
@@ -2666,3 +2669,43 @@ def select_sources(subject, label, location='center', extent=0.,
                           hemi=new_label.hemi, name=name, subject=subject)
 
     return new_label
+
+
+def find_label_in_annot(pos, subject='fsaverage', annot="aparc.a2005s+aseg",
+                         subjects_dir=None):
+    """
+    Find name in atlas for given MRI coordinates.
+
+    Parameters
+    ----------
+    pos : vector of x,y,z coordinates in MRI space
+    subject : MRI subject name
+    annot : parcellation file name
+    subjects_dir : path to MRI subjects directory
```

Comment:
see how we do parameters in our other public functions. First line of each entry is `param name : allowed type | other allowed type`. Second and subsequent lines are indented with description formatted like a proper sentence/paragraph.